### ISSUE TITLE
FusedBatchNorm mean/variable dimension is not correct after conversion 

### ISSUE THREAD

**Issue by pengwa** [onnx/tensorflow-onnx#98]
TF FusedBatchNorm(https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/fused-batch-norm)  is mapped to ONNX BatchNormalization(https://github.com/onnx/onnx/blob/master/docs/Operators.md#batchnormalization).

Though tf FusedBatchNorm document mentioned that mean/variable/offset/scale's size should be same as the channel number of input. But in observation of generated tf graph, mean/variable has size [0], even though scale/offset has [1024]. 

![image](https://user-images.githubusercontent.com/10530022/43466344-e7e730c4-9511-11e8-98b3-c36319b32f33.png)

![image](https://user-images.githubusercontent.com/10530022/43466401-07903aec-9512-11e8-9372-3a7ba324427a.png)

![image](https://user-images.githubusercontent.com/10530022/43466478-3031ca74-9512-11e8-9ee2-fe305990caad.png)

![image](https://user-images.githubusercontent.com/10530022/43466502-4019d4d6-9512-11e8-92df-545ab61a9f1d.png)

In ONNX, scale/bias/mean/var's size need to be equal to channel count, and MUST has same data type. 

Possible solution is: to extend tf FusedBatchNorm mean/var's size to be same of offset/scale, if their size is different.
